### DISCUSSION TITLE
.NET 7.0 Available

### DISCUSSION THREAD

**Discussion by DRA6N** [rbwhitaker/rbwhitaker.wikidot.com#11] [General]
I am working through your C# tutorial series and it says to let you know when .NET 7.0 framework is available. ... It's available :D

**Comment by rbwhitaker** [comment]
I've updated the tutorial to call out .NET 7. Thanks for reporting it! Sheesh. I didn't realize it had been this long since this came in. I _think_ I might have gotten a much more recent email or Discord message or something about it in the last two weeks or so. I usually do a better job at these things. Anyway, it is updated. For now. .NET 8 arrives in a little more than a month. :D
